tests/eas/load_tracking.py

```diff
@@ -41,6 +42,14 @@
 ERROR_MARGIN_PCT = 15
 # Allowed error margin % for migration tests
 MIGRATION_ERROR_MARGIN_PCT = 2
+# When running migration tests intererence from kthreads could skew the results
```

Comment:
This test is quite good... we should also probably add an explicit section in the documentation of each test class to clearly state what are the _expected invariants_ of its tests. In this case: we do not expect other tasks running on the target CPUs for a percentage of time bigger then the configured threshold.

Reply:
OK I'll update the test docs.